improver/utilities/mathematical_operations.py

```diff
@@ -356,3 +356,52 @@ def process(self, cube):
         integrated_cube = self.perform_integration(upper_bounds_cube, lower_bounds_cube)
 
         return integrated_cube
+
+
+def alinfit(X, Y, axis=-1, gradient_only=False):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Argument name "Y" doesn't conform to snake_case naming style](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)